python_hooks/dockerfile_poetry.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import re
+import sys
+
+REGEX_MATCH = r"poetry[~]?=[\d.]+"
+
+
+def check_poetry(filename):
+    with open(filename) as file:
+        dockerfile = file.read()
+
+    if not re.search(REGEX_MATCH, dockerfile):
+        print(f"Poetry version needs to be specified in {filename}")
+        return 1
+
+    return 0  # Indicates success
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Check for specific imports')
+    parser.add_argument('file_list', nargs='+', help='List of files to check')  # provided by the pre-commit call
```

Comment:
Where does `file_list` get passed? Is this supposed to be `files` (specified in `pre-commit-hooks.yaml`)?

Reply:
oh, right, the comment, nvm :D

Reply:
ooooone of these days... I'm gunna :git-gud: